app/components/collection-search.cjsx

```diff
@@ -23,7 +23,7 @@ module.exports = React.createClass
     query =
       page_size: 20
       favorite: false
-      current_user_roles: 'owner,collaborator,contributor,viewer'
+      current_user_roles: 'owner,collaborator,contributor'
```

Comment:
This makes sense because I added `viewer` in one commit and then removed it in a later commit when I noticed this `collection-search` was responsible for adding subjects to a collection. It should be kept as it currently it on master.